src/lib.rs

```diff
@@ -1643,3 +1643,42 @@ impl std::io::Seek for OutputReader {
         Ok(self.position())
     }
 }
+
+#[cfg(feature = "rand")]
+impl rand_core::SeedableRng for OutputReader {
+    type Seed = [u8; 32];
+
+    #[inline]
+    fn from_seed(seed: Self::Seed) -> Self {
+        let mut hasher = Hasher::new();
+        hasher.update(&seed);
```

Comment:
This is totally arbitrary and won't affect performance either way, but I have a slight preference for using `Hasher::new_keyed` with the seed as the key.

Reply:
I'm fine with that.  This one was just the easiest to quickly write tests to verify using `b3sum`.

Reply:
I've now switched this to use new_keyed, and updated the tests appropriately.